package.json

```diff
@@ -53,15 +53,14 @@
     "grunt-coveralls": "^1.0.1",
     "grunt-fh-build": "2.0.0",
     "istanbul": "^0.4.3",
-    "mocha": "^2.2.5",
+    "mocha": "^5.1.1",
     "nock": "^8.0.0",
     "proxyquire": "^1.7.4",
     "recursive-readdir": "^2.0.0",
     "sinon": "^1.17.3",
-    "sinon-as-promised": "4.0.2",
-    "turbo-test-runner": "0.3.3"
+    "sinon-as-promised": "4.0.2"
   },
   "engines": {
-    "node": "4.4"
+    "node": ">=6.11.3"
```

Comment:
is this to align with a specific version used by the studio? Otherwise maybe just `>=6` here?

Reply:
Yes, it is aligned, and the versions > 6 and < 6.11.3 should not be used because of CEVs. Also, the test here is with 6 which is  6.11.3.

Reply:
I believe this module is _only_ used by platform components (fh-supercore and fh-mbaas). I'd suggest going with what Damien suggested, based on https://github.com/fheng/fhcap/blob/FH-v3.20/wrapper-cookbooks/nodejs_feedhenry/attributes/default.rb